app/src/main/java/br/com/catbag/gifreduxsample/models/AppState.java

```diff
@@ -19,4 +19,9 @@
         return gifs;
     }
 
+    @Value.Default
+    public boolean getHasMoreGifs() {
+        return true;
```

Comment:
Remember that this is a initial state, and in the initial state we never have been fetched anything.